ws/ws.go

```diff
@@ -99,7 +100,6 @@ func Write(key *datastore.Key, buf []byte) error {
 }
 
 func reader(ws *websocket.Conn) {
-	defer ws.Close()
```

Comment:
I removed this because after a few runs nothing arrives in the frontend. We could close the ws programatically after some time.